### DISCUSSION TITLE
Not able to call jira api using service account to create bug/issue

### DISCUSSION THREAD

**Discussion by pkfubo** [andygrunwald/go-jira#644] [Q&A]
I am not able to create Bug/Issue. I am using a go jira sdk ([GitHub - andygrunwald/go-jira: Go client library for Atlassian Jira](http://github.com/andygrunwald/go-jira)) to create bug/issue.

Strange thing is when i use my personal account id which has format “df45g5541d8834b550c18e4g”, issue/bug is being successfully created but when i a service account which has id format as “s34fg5:bfd5ddh-bnerbr76556-bmgbegege667g-nmefge7987” it does not create the issue/bug. Although payload is same in both the cases.

error i get : “errorString {s: “request failed. Please analyze the request body for more details. Status code: 400”}”

this is the actual payload:

“{"fields":{"assignee":{"Password":"","accountId":"713030:a6a3dfa0-a7ac-56d4-f56b-f33377b9ee57"},"description":"Hello There is a new creative created","issuetype":{"name":"Bug"},"project":{"key":"VXAD"},"reporter":{"Password":"","accountId":"713030:a6a3dfa0-a7ac-56d4-f56b-f33377b9ee57"},"summary":"Jira Test"}}\n”

@andygrunwald

**Comment by andygrunwald** [comment]
@pkfubo Why are you not able to create a big/issue?
Did you try it via https://github.com/andygrunwald/go-jira/issues/new?assignees=&labels=&projects=&template=bug_report.md&title= ?

Please provide a bit information like

## What happened?

Please provide as much info as possible.
Not doing so may result in your bug not being addressed in a timely manner.

## What did you expect to happen?


## How can we reproduce it (as minimally and precisely as possible)?

Minimal code helps us to identify the problem faster.

## Anything else we need to know?


## Your Environment

Include as many relevant details about the environment you experienced the problem in

* go-jira version (git tag or sha):
* Go version (`go version`):
* Jira type (cloud or on-premise):
* Jira version / Api version:
